Gemfile

```diff
@@ -1,32 +1,26 @@
 source "https://rubygems.org"
 
-ruby '1.9.3'
+ruby '2.7.2'
```

Comment:
Remove ruby version from Gemfile.

Reply:
Just to know why don't you want to have the ruby version specified @blambeau ?